modules/ce_etherpad/ce_etherpad.install

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Implements hook_requirements().
+ */
+function ce_etherpad_requirements($phase) {
+  $requirements = [];
+
+  if ($phase == 'runtime') {
+    // Check for the client library.
+    $requirements['etherpad_client'] = [
+      'title' => t('Etherpad client library'),
+    ];
+    if (class_exists('EtherpadLite\Client')) {
+      $requirements['etherpad_client']['value'] = t('Installed');
+      $requirements['etherpad_client']['severity'] = REQUIREMENT_OK;
+    }
+    else {
+      $requirements['etherpad_client']['value'] = t('Not installed');
+      $requirements['etherpad_client']['description'] = t('Either Http Client library is not installed. Run "composer require 0x46616c6b/etherpad-lite-client" in root directory of Drupal to install Etherpad Http Client libreary.');
+      $requirements['etherpad_client']['severity'] = REQUIREMENT_ERROR;
+    }
+
+    // Check for the jquery plugin.
+    $requirements['etherpad_plugin'] = [
```

Comment:
The error is not showing if the etherpad_plugin is not installed

Reply:
I have re-check, this error is showing if etherpad jQuery plugin not exists

![Screenshot from 2019-06-25 16-54-14](https://user-images.githubusercontent.com/22341750/60094505-e9b7ae00-9769-11e9-962f-51b42566b0bc.png)

Reply:
It should throw an error on the same page i.e configuration page like it is throwing for the composer package

Reply:
At "Test Connection"?
In testing connection, composer package is necessary
jQuery plugin is used to load Etherpad

The reason why I don't prefer this error to be there
1. jQuery plugin is part of our Requirement, And Drupal has dedicated a place for requirement error So, I'm throwing REQUIREMENT_ERROR which you can see in the screenshot
2. Repetition of the same check at two places.

Reply:
At least show this error during plugin installation like a warning else user check when its plugin is not working and then he will see the logs.

Reply:
Implemented with commit 4c16164